test/integration/targets/azure_rm_virtualmachine/tasks/virtualmachine.yml

```diff
@@ -130,6 +130,21 @@
 - assert:
       that: not output.changed
 
+- name: Create an image from VM
```

Comment:
Can we have 3 tests per operation

* test with check mode to make sure an image wasn't actually created
* test normally (what is here right now)
* test again to check idempotency and make sure no change is recorded